packages/framework/src/Resources/styles/admin/layout/header/top-info.less

```diff
@@ -1,32 +1,31 @@
 .top-info {
```

Comment:
In this whole file, tabs are used instead of whitespaces. Why?

Reply:
It took a while for kulincz to configure his IDE, I thought I've fixed up everything... Fixed.